src/main/resources/assets/wurst/lang/ro

```diff
@@ -0,0 +1,166 @@
+  "description.wurst.hack.tillaura": "Transforma automat dirtul in teren agricol.\nA nu fi confundat cu KillAura.",
+  "description.wurst.hack.timer": "Schimba viteza la aproape tot.",
+  "description.wurst.hack.tired": "Te face sa arati ca si Alexander in 2015.\nVizibil numai pentru alti jucatori.",
+  "description.wurst.hack.toomanyhax": "Blocheaza orice feature nedorit.\nIti permite sa iti alegi hackurile si sa fi precaut pentru a nu-ti lua ban.\nPentru cei care vor  \"doar sa se distreze un pic\".\n\nFoloseste §6.toomanyhax§r pentu a alege ce optiuni sa blochezi.\nscrie §6.help toomanyhax§r pentru mai multe informatii.",
+  "description.wurst.hack.tp-aura": "Ataca automat cea mai apropiata entitate valida, teleportandu-se in jurul ei.",
+  "description.wurst.hack.trajectories": "Prezice traiectoria sagetilor si a itemelor aruncate.",
+  "description.wurst.hack.treebot": "Un bot experimental care se plimba si taie copaci.\nMomentan este limitat copacilor de dimensiune mica.",
+  "description.wurst.hack.triggerbot": "ataca automat entitatea la care te uiti .",
+  "description.wurst.hack.trollpotion": "Genereaza o potiune cu multe efecte enervante.",
+  "description.wurst.hack.truesight": "Iti permite sa vezi entitati invizibile.",
+  "description.wurst.hack.tunneller": "Sapa automat un tunel.\n\n§c§lWARNING:§r Desi acest bot va incerca sa nu moara de la fall damage sau lava, nu este garantat ca nu va muri. Foloseste-l numai cu echipament pe care esti dispus sa il pierzi.",
+  "description.wurst.hack.x-ray": "Iti permite sa vezi minereuri prin pereti.",
+  "description.wurst.altmanager.premium": "Acest cont alternativ are o parola si poate intra pe servere.",
+  "description.wurst.altmanager.cracked": "Acest cont alternativ nu are o parola si nu poate intra decat pe servere cracked.",
+  "description.wurst.altmanager.failed": "Ultima data cand ai incercat sa te loghezi cu acest cont alternativ, nu a functionat.",
+  "description.wurst.altmanager.checked": "Parola a functionat in trecut.",
+  "description.wurst.altmanager.unchecked": "Nu ai reusit niciodata sa te loghezi in acest cont alternativ.",
+  "description.wurst.altmanager.favorite": "Ai marcat acest cont alternativ la favorite.",
+  "description.wurst.altmanager.window": "Acest buton deschide o alta fereastra.",
+  "description.wurst.altmanager.window_freeze": "It might look like the game is not responding while that window is open.",
+  "description.wurst.altmanager.fullscreen": "§cOpreste modul fullscreen!",
+  "gui.wurst.altmanager.folder_error.title": "folderul'.Wurst encryption' nu a putut fi creeat!",
+  "gui.wurst.altmanager.folder_error.message": "Este posibil ca tu sa fi luat accesul instalatiei Wurst de la acest folder.\nAltManager cannot encrypt or decrypt your alt list without it.\nYou can still use AltManager, but any alts you create now won't be saved.\n\nThe full error is as follows:\n%s",
```

Comment:
This is partially still in English.

Reply:
The last part cannot be properly translated as there are no words in Romanian that go well with them. If I were to translate that last bit the error would not make sense and would be very misleading. I'll still try if you'd like me to.